### DISCUSSION TITLE
The realm which should be first visible after logging in as an admin in Keycloak is not changing. It is master.

### DISCUSSION THREAD

**Discussion by ankita2402** [keycloak/keycloak#29760] [Archive]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/cli

### Describe the bug

I want the realm that I am creating should be first visible when I login in as an admin. I am first getting the master realm after logging in and then I have to change the realm from the drop down on the left, which is the expected behavior.
But I want to have my personal realm to be visible first when logging in as an admin. 
Can someone please share steps from where I can change this configuration ? Is it the themes where we change that or somewhere else ?

### Version

24.0.2

### Regression

- [ ] The issue is a regression

### Expected behavior

My realm should be first visible after logging in as an admin in Keycloak.

### Actual behavior

Master realm is visible first, and after that I have to change the realm from dropdown manually. 

### How to Reproduce?

Always reproducible.

### Anything else?

I tried changing the admin url in the index.ftl of the themes which I am loading, but this is not working.

**Comment by ankita2402** [comment]
Any updates on this ??

**Comment by guptakaran27** [comment]
Hi Team, 
I am also facing a similar issue in Keycloak. 
Any properties file that can be over-written for this?
